apps/website/src/data/tech/commands.ts

```diff
@@ -133,6 +133,62 @@ const commands: Record<string, Command> = {
       },
     ],
   },
+  ptzseta: {
+    description:
+      "Move to coordinates, Control auto-focus (if supported), Change absolute focus (if supported)",
```

Comment:
I think we should keep referring to it by pan/tilt/zoom instead of coordinates since we use that for the other descriptions.

```suggestion
      "Change absolute pan/tilt/zoom, control auto-focus (if supported), change absolute focus (if supported)",
```